frontend/talentsearch/src/js/components/skills/AddedSkills/AddedSkills.tsx

```diff
@@ -19,25 +19,29 @@ const AddedSkills: React.FunctionComponent<AddedSkillsProps> = ({
   const locale = getLocale(intl);
   return (
     <>
-      <h5>
+      <h3 data-h2-font-size="b(h5)">
         {intl.formatMessage({
           defaultMessage: "Skills attached to this experience",
           description:
             "Section header for a list of skills attached to this experience",
         })}
-      </h5>
-      {skills.map((skill) => {
-        const handleDismiss = () => onRemoveSkill(skill.id);
-        return (
-          <Chip
-            key={skill.id}
-            label={skill.name?.[locale] ?? "Missing Name"}
-            color="neutral"
-            mode="outline"
-            onDismiss={handleDismiss}
-          />
-        );
-      })}
+      </h3>
+      {skills.length ? (
+        <Chips>
+          {skills.map((skill) => {
+            const handleDismiss = () => onRemoveSkill(skill.id);
+            return (
+              <Chip
+                key={skill?.id}
```

Comment:
I'm getting a
```
 Warning: Each child in a list should have a unique "key" prop.
Check the render method of `AddedSkills`.
```
on the skill edit page.  Is it because this is possibly null?

Reply:
So, this is the annoying bit. I updated this,`skill.id -> skill?.id` because when testing, we generate mock skills and experiences separately. This results in the `id` for skills not matching the `id` for skills attached to experiences.

It was actually breaking because when we pass in the skills that are added, they don't match the available skills and we get a "cannot access property `id` of `undefined`. This breaks the tests, adding the `?.id` throws some weird errors but does not break the tests.

I think we should create a ticket to update the `fakeExperiences` to allow us to pass in valid skills to be used in the generation. Once that is updated, we can revert this back to remove the warning.